src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts

```diff
@@ -842,7 +842,7 @@ export class AssessmentPlayerComponent implements OnInit, OnDestroy, ComponentCa
             if (response && response.context) {
               response.context.objectRollup = this.objectRollUp;
             }
-            const contentDetails = {contentId: id, contentData: response.questionSet };
+            const contentDetails = {contentId: id, contentData: response.questionset };
```

Comment:
please use the ? operator as if response is null the code fails

Reply:
added the ? operator